pieman/pieman/util.py

```diff
@@ -32,6 +53,21 @@ def connect_to_redis(host, port):
     return conn
 
 
+def download(url, dst, quiet=False):
```

Comment:
Downloads the specified document from the Web, (optionally) displaying a progress bar.